src/lib/db/index.ts

```diff
@@ -14,6 +14,28 @@ export interface Models<V> {
 
 export type Query = Knex.QueryBuilder
 
+const demandColumns = [
+  'id',
+  'owner',
+  'state',
+  'subtype',
+  'parameters_attachment_id AS parametersAttachmentId',
+  'latest_token_id AS latestTokenId',
```

Comment:
this is where I hope that we could get to the point where we have models like in prisma -> https://www.prisma.io/docs/concepts/components/prisma-schema/data-model, However, wanted to find some knex references but was not able to validate, so far got this -> https://knexjs.org/guide/#postprocessresponse. [MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller)

Reply:
yes the case swapping is not ideal